transcom/mymove
# [MB-3316] TIO shown payment requests on invoicing queue page

## Description

This moves the payment requests index route for the TIO user to be displayed after login and at the invoicing/queue route.

In the e2e basic scenario seed data, I added more service items to the existing payment request record in the database.  I've also created 2 new payment request records to test with that are both associated with the same move task order.  In the past we were only creating mto_service_items but now we need to add payment_service_items now that include a price.

Also added logic to remove the headers when visiting the document viewer routes.  This includes viewing the move orders as well as reviewing the service items.  These pages are supposed to forego displaying the main header and txo navbar.

## Reviewer Notes

I had to pull in changes from Moncef's move orders database cleanup otherwise I couldn't make changes to the generate-test-data tool which was temporarily broken.

If there is rate pricing logic for service items that would make more sense please let me know, it got a little complicated so I ended up using placeholder values. 

## Setup

```
// you may want to clean the old binary build tools and recompile
rm bin/generate-test-data
make build_tools
// destroys and recreates dev db w/ migrations and loads the e2d basic scenario in development
make db_dev_e2e_populate
make server_run
make office_client_run
```

1. Sign in as the TIO user
2. The first page loaded should now be the list of all payment requests (currently only 1)
3. Confirm that `officelocal:3000/invoicing/queue` takes you to the same page.
4. Clicking on the payment request link should display the payment request index page showing approve or reject actions.  This has been changed to the review service items page that should be merged shortly.
5. Notice the headers don't show

## Code Review Verification Steps

 [Querying the Database Safely](https://github.com/transcom/mymove/wiki/backend#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:

## References

* [Jira story](tbd) for this change
* [this article](tbd) explains more about the approach used.

## Screenshots

![Screen Shot 2020-07-21 at 1 13 19 PM](https://user-images.githubusercontent.com/52669884/88085652-2532d080-cb54-11ea-8bb0-f7242ba25f97.png)
![Screen Shot 2020-07-21 at 1 12 36 PM](https://user-images.githubusercontent.com/52669884/88085678-2cf27500-cb54-11ea-98f4-3f60f50979c2.png)
![Screen Shot 2020-07-21 at 1 14 11 PM](https://user-images.githubusercontent.com/52669884/88085689-2fed6580-cb54-11ea-9deb-d8f99c204b02.png)
